### PR TITLE
DropDown width issues solved

### PR DESCRIPTION
**Pull request by boutinb**
Check test DropDown in jaspTestModule

The fieldWidth property gives the width of the internal dropdown control. If this property is set, it fixes the width. This feature is needed, especially when DropDown controls are used in a ComponentList, where the width of the controls should not changed with the values of the dropdown.

This is also needed to solves some problem in PR https://github.com/jasp-stats/jaspProcess/pull/108

### CODE REVIEW

File: QMLComponents/components/JASP/Controls/ComboBox.qml [hunk 1]
```diff
@@ -19,59 +19,19 @@ ComboBoxBase
 	property alias	currentLabel:			comboBox.currentText
 	property alias	value:					comboBox.currentValue
 	property alias	indexDefaultValue:		comboBox.currentIndex
-	property alias	fieldWidth:				control.width
+	property alias	fieldWidth:				control.implicitWidth
+	property alias  controlText:			controlText
```
Comment (JorisGoosen) [inline QMLComponents/components/JASP/Controls/ComboBox.qml]:
```
	property alias  controlText:			controlText
```
???

Comment (JorisGoosen) [inline QMLComponents/components/JASP/Controls/ComboBox.qml]:
```suggestion
	property alias  controlText:			control.text
```

Maybe you mean this?

File: QMLComponents/controls/comboboxbase.h [hunk 2]
```diff
@@ -80,10 +80,10 @@ protected slots:
 	void activatedSlot(int index);
 
 	GENERIC_SET_FUNCTION(StartValue,	_startValue,	startValueChanged,	QString	)
-	GENERIC_SET_FUNCTION(FixedWidth,	_fixedWidth,	fixedWidthChanged,	bool	)
 
 protected:
 	bool _checkLevelsConstraints()			override;
+	void _setLongestValue();
```
Comment (JorisGoosen) [inline QMLComponents/controls/comboboxbase.h]: ???
```suggestion
```

**Review by JorisGoosen — CHANGES_REQUESTED** [review]
You see them now then?

File: QMLComponents/components/JASP/Controls/ComboBox.qml (hunk 1 above)
Comment (boutinb) [reply QMLComponents/components/JASP/Controls/ComboBox.qml]: I forgot to remove this

Comment (boutinb) [reply QMLComponents/components/JASP/Controls/ComboBox.qml]: No this property must be removed. `value` is already the value of the DropDown.

Comment (JorisGoosen) [reply QMLComponents/components/JASP/Controls/ComboBox.qml]: So remove it then

**Review by shun2wang — APPROVED** [review]
DropDown now works well for me.

**Comment by boutinb** [comment]
Should we merge it @JorisGoosen ?

**Pull request merged**